core/services/ocr2/plugins/s4/config.go

```diff
@@ -1,6 +1,7 @@
 package s4
 
 type PluginConfig struct {
+	DONID                   string
```

Comment:
I was wrong when I told you that this is a job spec config. This actually needs to go onchain and be translated to/from protos. For simplicity I suggest to remove it for now. In current deployments one node supports a single DON anyway. When that changes we might have other domain separators coming from LOOPPs.